src/battle_ai_main.c

```diff
@@ -2654,6 +2656,23 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
             return 0;   // cannot even select
     } // move effect checks
 
+    // Choice items
+    if (gBattleMons[battlerAtk].ability != ABILITY_KLUTZ
```

Comment:
Small nitpick, but I would change the order here, and first check holdEffect, then ability